cdap-docs/developers-manual/source/building-blocks/metadata-lineage.rst

```diff
@@ -23,8 +23,8 @@ CDAP captures metadata from many different sources |---| as well as those specif
 user |---| on different entities and objects. The container model of CDAP provides for the
 seamless aggregation of a wide variety of machine-generated metadata that is automatically
 associated with datasets. This capability allows flexibility for the developers and data
-scientist to innovate and build solutions on Hadoop without any responsibility for
-compliance and governance.
+scientist to innovate and build solutions on Hadoop, while simultaneously having a lineage
```

Comment:
'having a lineage' does not sound right. perhaps 'without having to worry about maintaining compliance and governance in every application'